app/backend/wells/migrations/0070_ems_id.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.8 on 2019-04-08 15:57
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wells', '0069_auto_20190405_2220'),
+    ]
+
+    operations = [
```

Comment:
Isn't the column already gone at this point? It would have been renamed in migration 0069